tasks/fh-build.js

```diff
@@ -381,7 +381,7 @@ module.exports = function(grunt) {
     } else if (this.target === 'shrinkwrap') {
       grunt.task.run(['shell:fh-run-array:fhshrinkwrap']);
     } else if (this.target === 'default') {
-      grunt.task.run(['jshint', 'fh:coverage', 'fh:analysis', 'fh:dist']);
+      grunt.task.run(['jshint', 'fh:dist']);
```

Comment:
The tests should still be run as part of default PR job, so instead of `fh:coverage`, we should now have `fh:test` here I think.